app/config.py

```diff
@@ -101,43 +102,40 @@ def get_abs_path(file_path: str):
 # Useful when calling Postfix from an external network
 POSTFIX_SUBMISSION_TLS = "POSTFIX_SUBMISSION_TLS" in os.environ
 
-if "OTHER_ALIAS_DOMAINS" in os.environ:
```

Comment:
Hey unfortunately this value is set with `[` ... e.g. `["domain1", "domain2"]` so this change will create a regression

Reply:
You're right, current implementation will introduce backward incompatibility. I'm going to write a simple utility function to handle